packages/backend-test-utils/src/next/wiring/TestBackend.test.ts

```diff
@@ -307,7 +307,7 @@ describe('TestBackend', () => {
         ],
       }),
     ).rejects.toThrow(
-      "Extension point registered for plugin 'testA' may not be used by module for plugin 'testB'",
+      "Illegal dependency: Module 'test' for plugin 'testB' attempted to depend on extension point 'a' for plugin 'testA'. Extension points can only be used within their plugin's scope.",
```

Comment:
This test felt a bit less intuitive as there was no `registerExtensionPoint` and therefore, the extension point does not really "belong" to any plugin. However, the `BackendInitializer` uses the `pluginId` of the first use of it (feature registration) as the `pluginId` of the extension point.

Hence, the extension point "a" belongs to plugin "testA".